woocommerce/woocommerce
# Remove `memoize-one` from woo admin & update it to ^6.0.0 for woo components

### All Submissions:






### Changes proposed in this Pull Request:



Closes #32884.

- Remove `memoize-one` from woo admin since we don't really use it in ./client.
- Update it to ^6.0.0 for woo components

### Breaking Changes
- This release is a major, but there are no behaviour or API changes. The major is to reflect that some of the TypeScript types have been tightened which might cause some peoples TypeScript builds to break.
https://github.com/alexreardon/memoize-one/releases

### How to test the changes in this Pull Request:

1. Run `pnpm nx build woocommerce-admin`
2. Go to `Analytics > Overview`
3. Confirm that all charts should be rendered without errors.

### Other information:




### FOR PR REVIEWER ONLY:
